docs/usage.md

```diff
@@ -170,69 +212,106 @@ params {
 }
 ```
 
-<!-- TODO nf-core: Describe reference path flags -->
 ### `--fasta`
+
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
 --fasta '[path to Fasta reference]'
 ```
 
+> Note that you need to specify either a `--genome` or `--fasta` when running the STARsolo workflow. The Kallisto and Alevin workflows can utilize a `--transcript_fasta` instead, whereas STAR needs a genomic FastA file as input in all cases.
```

Comment:
```suggestion
> Note that you need to specify either a `--genome` or `--fasta` when running the STARsolo workflow. The Kallisto and Alevin workflows can utilize a `--transcript_fasta` instead, whereas STAR needs a genomic fasta file as input in all cases.
```

Reply:
isn't it fasta files? or is it actually FastA files? Have I been lied to my whole life?